apps/dashboard/src/components/settings/Account/Billing/Pricing.tsx

```diff
@@ -38,108 +61,53 @@ export const BillingPricing: React.FC<BillingPricingProps> = ({
     }, 1000);
   });
 
-  const starterCta: CtaLink | undefined = useMemo(() => {
-    switch (validTeamPlan) {
-      // free > starter
-      case "free": {
-        return {
-          label: "Get started for free",
-        };
-      }
-
-      // starter > starter
-      case "starter": {
-        return undefined;
-      }
-
-      // growth > starter
-      case "growth": {
-        return {
-          label: "Downgrade",
-        };
-      }
-
-      // pro > starter
-      case "pro": {
-        return {
-          label: "Contact us",
-          target: "_blank",
-        };
-      }
-    }
-  }, [validTeamPlan]);
-
-  const growthCardCta: CtaLink | undefined = useMemo(() => {
-    switch (validTeamPlan) {
-      // free > growth
-      case "free": {
-        return {
-          label: "Get started",
-        };
-      }
-
-      // starter > growth
-      case "starter": {
-        return {
-          label: "Upgrade",
-        };
-      }
-
-      // growth > growth
-      case "growth": {
-        return undefined;
-      }
-
-      // pro > growth
-      case "pro": {
-        return {
-          label: "Contact us",
-          target: "_blank",
-        };
-      }
-    }
-  }, [validTeamPlan]);
+  const starterCta = getPlanCta(validTeamPlan, "starter");
+  const growthCta = getPlanCta(validTeamPlan, "growth");
+  const accelerateCta = getPlanCta(validTeamPlan, "accelerate");
+  const scaleCta = getPlanCta(validTeamPlan, "scale");
 
-  const proCta: CtaLink | undefined = useMemo(() => {
-    // pro > pro
-    if (validTeamPlan === "pro") {
-      return undefined;
-    }
+  const highlightGrowthPlan =
```

Comment:
if the plan is `growth` we should highlight `accelerate`, if the plan is `accelerate` we should highlight `scale`

Reply:
Updated